lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -191,3 +193,22 @@ def psfont(*args, **kwargs):
     ax.text(0.5, 0.5, 'hello')
     with tempfile.TemporaryFile() as tmpfile, pytest.raises(ValueError):
         fig.savefig(tmpfile, format='pdf')
+
+
+def test_pdf_savefig_when_color_is_none():
+    backup_params = mpl.rcParams.copy()
+    mpl.rcParams.update(mpl.rcParamsDefault)
```

Comment:
I will have to get back on this one when I'm back from work. It was two days ago, can't recall the reason on the spot.